test/pxutils.test.js

```diff
@@ -1,10 +1,10 @@
 'use strict';
```

Comment:
Looks good, what about testing support configurable graphql paths

Reply:
I will add it to e2e tests

Reply:
https://github.com/PerimeterX/enforcer_specs_tests/pull/136